docs/Configuration/Configuration.md

```diff
@@ -0,0 +1,149 @@
+﻿# KMD Logic Document Generation Configuration
+
+To configure a Logic Document Generation environment, you need to access [Logic Console](https://console.kmdlogic.io/) to do so.
+
+## Accessing Logic Document Generation Configuration module
+
+To access the Logic Document Generation Configuration module you will need to click on the menu labeled as `Document Generation` on the sidebar menu (highlighted in the image below).  
+
+![Image of Logic Console sidebar menu](./assets/image001.jpg)
+
+After doing that you will be presented with the `Logic Document Generation Configuration` module.
+
+![Image of Logic Console Document Generation Configuration module](./assets/image002.jpg)
+
+
+## Creating a Configuration environment
+
+To create a Document Generation Configuration click on the `ADD CONFIGURATION` button at the top right corner of the screen.
+
+You will be then presented with the following screen:
+
+![Image of Logic Console Create Document Generation Configuration screen](./assets/image003.jpg)
+
+The screen is divided in two sections, the `Configuration Settings` and the `Master Storage Settings`, we will explain them below in detail.
+
+### Configuration Settings
+
+![Image of Logic Console Create Document Generation Configuration - Settings](./assets/image004.jpg)
+
+- Configuration name:  
+_(Required)_ The configuration name, any non-empty string is valid.
+
+- ASPOSE license switch:  
+_(Required)_ In order to use `Logic Document Generation` your organization needs to have a valid ASPOSE license. Toggle this switch on to acknowledge that.
```

Comment:
probably should say team rather than organisation

Reply:
Fixed in 3e485059348ca4959c60435516832dab8ec7c166